Assets/Scenes/TutorialLevel.unity

```diff
@@ -4889,15 +4889,15 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 1378278090585045583, guid: 901e6b7f6ba08daf59f185b6868a2d12, type: 3}
       propertyPath: m_LocalPosition.x
-      value: 2.7779694
+      value: 3.5
```

Comment:
I believe this is the player being moved right? And I assume you set the parent to 0,0,0 too then?

Reply:
yes